sdcm/cluster.py

```diff
@@ -3421,6 +3422,12 @@ def terminate_node(self, node: BaseNode, scylla_shards: int = 0) -> None:
             ))
         if node in self.nodes:
             self.nodes.remove(node)
+        ScyllaLogCollector.cluster_log_type = node.name
```

Comment:
I'm not sure what this is trying todo ... have a unique type ? I'm quite sure we have a list somewhere of the types, and this won't fit.